test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -82,21 +82,6 @@
         "conversionDate": "number",
         "conversionRate": 1700,
         "usdConversionRate": 1700
-      },
-      "SepoliaETH": {
-        "conversionDate": "number",
-        "conversionRate": 1700,
-        "usdConversionRate": 1700
-      },
-      "LineaETH": {
-        "conversionDate": "number",
-        "conversionRate": 1700,
-        "usdConversionRate": 1700
-      },
-      "MegaETH": {
-        "conversionDate": "number",
-        "conversionRate": 1700,
-        "usdConversionRate": 1700
```

Comment:
It is interested to see that these are not added to the background state snapshot.
This is because we are filtering out these networks in the `useCurrencyRatePolling.ts` file.

This was originally Salims change, does it seem right that we are skipping these networks?